src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -15,10 +15,20 @@ public class ConsoleLogger : ILogger
         private static readonly object _lock = new object();
         private static readonly string _loglevelPadding = ": ";
         private static readonly string _messagePadding;
+        private static readonly string _newLineWithMessagePadding;
 
         // ConsoleColor does not have a value to specify the 'Default' color
         private readonly ConsoleColor? DefaultConsoleColor = null;
 
+        // do not change user's background color except for Critical
+        private readonly ConsoleColors CriticalConsoleColors = new ConsoleColors(ConsoleColor.White, ConsoleColor.Red);
+        private readonly ConsoleColors ErrorConsoleColors = new ConsoleColors(ConsoleColor.Red, ConsoleColor.Black);
```

Comment:
@BrennanConroy 

> ConsoleColor.Black != user's background color

That's what was there before. I just made them instance members instead of producing a new ConsoleColors instance each time it was needed. (see [here](https://github.com/aspnet/Logging/blob/3d853c72eb9fb7e6182840d5c7811769dd431c62/src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs#L239))

> Oh, I think the comment is outdated, we made a decision to set black as background for everything to make sure the foreground color was visible

Idem. This comment was simply moved with the colors. (see [here](https://github.com/aspnet/Logging/blob/3d853c72eb9fb7e6182840d5c7811769dd431c62/src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs#L233))